### PR TITLE
Add tr() to visible strings; fix #119

### LINKED ISSUE #119: make iaito translatable
This is really easy. Just change every `"string"` to `tr("string")` and iaito can be translated. Out of experience the sooner this is done the easier it is. Once the code is bigger it is really annoying to do.

### PR DESCRIPTION
**Pull request by iwiwsb**
I changed every user-visible `"string"` to either `tr("string")` or `QObject::tr("string")`. Hope I haven't missed any.

### CODE REVIEW

**Comment by xarkes** [comment]
Hmmm it seems Travis build is broken. Also if you did them all, add "Fix #xxx" to your commit with XXX referencing the issue talking with tr.

**Comment by iwiwsb** [comment]
Done. Sorry for so many title changes.

**Comment by xarkes** [comment]
It does not compile on my machine:
```
mainwindow.cpp:370:84: error: no matching function for call to ‘MainWindow::tr(const QString)’
                              "\n" + core->cmd("ie") + "\n" + core->cmd("iM") + "\n"));
```
```
mainwindow.cpp:943:67: error: cannot convert ‘QString’ to ‘const char*’ for argument ‘1’ to ‘int printf(const char*, ...)’
     printf(tr("debug output: %s\n"), msg.toLocal8Bit().constData());
```

File: src/mainwindow.cpp [hunk 1]
```diff
@@ -940,7 +940,7 @@ void MainWindow::addOutput(const QString &msg)
 
 void MainWindow::addDebugOutput(const QString &msg)
 {
-    printf("debug output: %s\n", msg.toLocal8Bit().constData());
+    printf(tr("debug output: %s\n"), msg.toLocal8Bit().constData());
```
Comment (xarkes) [inline src/mainwindow.cpp]: No need to translate this debug message.

File: src/mainwindow.cpp [hunk 2]
```diff
@@ -963,11 +963,11 @@ void MainWindow::on_actionRun_Script_triggered()
     dialog.setDirectory(QDir::home());
 
     QString fileName;
-    fileName = dialog.getOpenFileName(this, "Select radare2 script");
+    fileName = dialog.getOpenFileName(this, tr("Select radare2 script"));
     if (!fileName.length()) //cancel was pressed
         return;
 
-    qDebug() << "Meow: " + fileName;
+    qDebug() << tr("Meow: ") + fileName;
```
Comment (xarkes) [inline src/mainwindow.cpp]: No need to translate.

File: src/qrdisasm.cpp [hunk 3]
```diff
@@ -15,6 +15,6 @@ bool IaitoRDisasm::disassembleAt(ut64 addr, IaitoRDisasmOption opt, IaitoRDisasm
     IAITONOTUSED(opt);
     IAITONOTUSED(dr);
 
-    printf("FUCK\n");
+    printf(QObject::tr("FUCK\n"));
```
Comment (xarkes) [inline src/qrdisasm.cpp]: No need to translate.

File: src/createnewdialog.cpp [hunk 4]
```diff
@@ -119,17 +119,17 @@ void createNewDialog::on_buttonCreate_clicked()
...
         else
         {
-            __alert("Invalid file size");
+            __alert(tr("Invalid file size"));
         }
     }
     else if (type == "C Code")
     {
-        __alert("C Code: TODO");
+        __alert(tr("C Code: TODO"));
```
Comment (xarkes) [inline src/createnewdialog.cpp]: No need to translate a TODO message

File: src/createnewdialog.cpp [hunk 5]
```diff
@@ -174,12 +174,12 @@ void createNewDialog::on_buttonCreate_clicked()
             }
             else
             {
-                __alert("failed to open file");
+                __alert(tr("failed to open file"));
```
Comment (xarkes) [inline src/createnewdialog.cpp]: Missing capital letter

File: src/createnewdialog.cpp [hunk 6]
```diff
@@ -199,24 +199,24 @@ void createNewDialog::on_buttonCreate_clicked()
...
```
Comment (xarkes) [inline src/createnewdialog.cpp]: Capital F

File: src/createnewdialog.cpp [hunk 7]
```diff
@@ -199,24 +199,24 @@ void createNewDialog::on_buttonCreate_clicked()
...
     {
-        __alert("Unknown combo value selected");
+        __alert(tr("Unknown combo value selected"));
         return;
     }
 
     if (format != "Raw")
     {
-        __alert("TODO: non-raw fileformat is not yet supported");
+        __alert(tr("TODO: non-raw fileformat is not yet supported"));
```
Comment (xarkes) [inline src/createnewdialog.cpp]: No need to translate TODO

**Review by xarkes — CHANGES_REQUESTED** [review]
Please fix this

**Comment by xarkes** [comment]
Cool thanks

**Pull request merged**